controllers/goharbor/core/deployments.go

```diff
@@ -30,34 +30,54 @@ var (
 	fsGroup    int64 = 10000
 	runAsGroup int64 = 10000
 	runAsUser  int64 = 10000
+
+	metricNamespace = "harbor"
+	metricSubsystem = "core"
```

Comment:
Are these constant not accessible from harbor repository?
I suggest to import them if possible so next update will be easier using go get -u`

Reply:
These constants are not accessible from the harbor repository.